ERROR StatusLogger Unable to locate a logging implementation, using SimpleLogger
2024-04-26 14:10:10,139 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,265 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,265 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,268 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`date` TIMESTAMP ) 
2024-04-26 14:10:10,284 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`date` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,284 [TRACE] BaseMappedStatement insert arguments: [2024-04-26 14:10:10.269]
2024-04-26 14:10:10,286 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,288 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,288 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,288 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`date` TIMESTAMP ) 
2024-04-26 14:10:10,289 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`date` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,289 [TRACE] BaseMappedStatement insert arguments: [2024-04-26 14:10:10.0]
2024-04-26 14:10:10,309 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,309 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
2024-04-26 14:10:10,313 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo` WHERE (`date` = ?)' with 1 args
2024-04-26 14:10:10,313 [TRACE] BaseMappedStatement prepared statement arguments: [2024-04-26 14:10:10.0]
2024-04-26 14:10:10,314 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo` WHERE (`date` = ?)' with 1 args returned 1 results
2024-04-26 14:10:10,314 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo` WHERE (`date` = ?)' with 1 args
2024-04-26 14:10:10,314 [TRACE] BaseMappedStatement prepared statement arguments: [2024-04-26 14:10:10.0]
2024-04-26 14:10:10,314 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo` WHERE (`date` = ?)' with 1 args returned 1 results
2024-04-26 14:10:10,315 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,317 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,317 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,317 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`date` TIMESTAMP ) 
2024-04-26 14:10:10,318 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`date` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,318 [TRACE] BaseMappedStatement insert arguments: [null]
2024-04-26 14:10:10,319 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,319 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
2024-04-26 14:10:10,320 [ERROR] LogbackLogBackendTest message
2024-04-26 14:10:10,320 [ERROR] LogbackLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testErrorStringThrowable(BaseLogBackendTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,322 [WARN] LogbackLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testWarnStringThrowable(BaseLogBackendTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,323 [INFO] LogbackLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testInfoStringThrowable(BaseLogBackendTest.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,323 [ERROR] LogbackLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testFatalStringThrowable(BaseLogBackendTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,324 [ERROR] LogbackLogBackendTest message
2024-04-26 14:10:10,324 [INFO] LogbackLogBackendTest message
2024-04-26 14:10:10,324 [INFO] LogbackLogBackendTest hello there
2024-04-26 14:10:10,325 [WARN] LogbackLogBackendTest message
2024-04-26 14:10:10,329 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,331 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,331 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,332 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`string` TEXT ) 
2024-04-26 14:10:10,333 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`string` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,333 [TRACE] BaseMappedStatement insert arguments: [str]
2024-04-26 14:10:10,334 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,335 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
2024-04-26 14:10:10,335 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo` WHERE (`string` = 'str')' with 0 args
2024-04-26 14:10:10,337 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo` WHERE (`string` = 'str')' with 0 args returned 1 results
2024-04-26 14:10:10,338 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo` WHERE (`string` = ?)' with 1 args
2024-04-26 14:10:10,338 [TRACE] BaseMappedStatement prepared statement arguments: [str]
2024-04-26 14:10:10,338 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo` WHERE (`string` = ?)' with 1 args returned 1 results
ERROR Log4j2LogBackendTest message
FATAL Log4j2LogBackendTest message java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testFatalStringThrowable(BaseLogBackendTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)

FATAL Log4j2LogBackendTest message
ERROR Log4j2LogBackendTest message java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testErrorStringThrowable(BaseLogBackendTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)

2024-04-26 14:10:10,347 [INFO] TableUtils dropping table 'lazy'
2024-04-26 14:10:10,349 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LAZY" not found; SQL statement:
DROP TABLE `lazy`  [42102-200]' for statement: DROP TABLE `lazy` 
2024-04-26 14:10:10,349 [INFO] TableUtils creating table 'lazy'
2024-04-26 14:10:10,361 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `lazy` (`id` INTEGER AUTO_INCREMENT , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,363 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `lazy` () VALUES ()' and 0 args, changed 1 rows
2024-04-26 14:10:10,363 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Lazy object
2024-04-26 14:10:10,363 [INFO] TableUtils dropping table 'foreign'
2024-04-26 14:10:10,364 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOREIGN" not found; SQL statement:
DROP TABLE `foreign`  [42102-200]' for statement: DROP TABLE `foreign` 
2024-04-26 14:10:10,364 [INFO] TableUtils creating table 'foreign'
2024-04-26 14:10:10,364 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foreign` (`id` INTEGER AUTO_INCREMENT , `lazy_id` INTEGER , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,365 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foreign` (`lazy_id` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,365 [TRACE] BaseMappedStatement insert arguments: [1]
2024-04-26 14:10:10,365 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foreign object
2024-04-26 14:10:10,367 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `lazy` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,367 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,367 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foreign` WHERE `lazy_id` = ?' with 1 args
2024-04-26 14:10:10,367 [TRACE] BaseMappedStatement prepared statement arguments: [1]
2024-04-26 14:10:10,367 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foreign` WHERE `lazy_id` = ?' with 1 args
2024-04-26 14:10:10,368 [TRACE] BaseMappedStatement prepared statement arguments: [1]
2024-04-26 14:10:10,368 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foreign` WHERE `lazy_id` = ?' with 1 args
2024-04-26 14:10:10,368 [TRACE] BaseMappedStatement prepared statement arguments: [1]
2024-04-26 14:10:10,370 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,379 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,379 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,380 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,383 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,383 [TRACE] BaseMappedStatement insert arguments: [13131511, 0, null]
2024-04-26 14:10:10,383 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,385 [DEBUG] BaseMappedStatement delete data with statement 'DELETE FROM `foo` WHERE `id` = ?' and 1 args, changed 1 rows
2024-04-26 14:10:10,385 [TRACE] BaseMappedStatement delete arguments: [1]
2024-04-26 14:10:10,386 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,386 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,386 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,386 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,398 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,400 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,400 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,401 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,403 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,403 [TRACE] BaseMappedStatement insert arguments: [0, 0, null]
2024-04-26 14:10:10,403 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,404 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,404 [TRACE] BaseMappedStatement insert arguments: [0, 0, null]
2024-04-26 14:10:10,404 [DEBUG] BaseMappedStatement assigned id '2' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,405 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,405 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 0 results
2024-04-26 14:10:10,406 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,407 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,408 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,409 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,409 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,410 [TRACE] BaseMappedStatement insert arguments: [13131511, 0, null]
2024-04-26 14:10:10,410 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,410 [DEBUG] BaseMappedStatement delete data with statement 'DELETE FROM `foo` WHERE `id` = ?' and 1 args, changed 1 rows
2024-04-26 14:10:10,410 [TRACE] BaseMappedStatement delete arguments: [1]
2024-04-26 14:10:10,410 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,410 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,411 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,411 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,416 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,417 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,417 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,418 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,418 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,418 [TRACE] BaseMappedStatement insert arguments: [13131511, 0, null]
2024-04-26 14:10:10,418 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,419 [DEBUG] BaseMappedStatement delete data with statement 'DELETE FROM `foo` WHERE `id` = ?' and 1 args, changed 1 rows
2024-04-26 14:10:10,419 [TRACE] BaseMappedStatement delete arguments: [1]
2024-04-26 14:10:10,419 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,419 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,419 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,419 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,421 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,422 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,422 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,423 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,423 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,423 [TRACE] BaseMappedStatement insert arguments: [0, 0, null]
2024-04-26 14:10:10,423 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,424 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,424 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,425 [DEBUG] BaseMappedStatement delete data with statement 'DELETE FROM `foo` WHERE `id` = ?' and 1 args, changed 1 rows
2024-04-26 14:10:10,425 [TRACE] BaseMappedStatement delete arguments: [1]
2024-04-26 14:10:10,426 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,426 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,426 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,427 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,427 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,428 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,429 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,429 [TRACE] BaseMappedStatement insert arguments: [13131511, 0, null]
2024-04-26 14:10:10,429 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,429 [DEBUG] BaseMappedStatement delete data with statement 'DELETE FROM `foo` WHERE `id` = ?' and 1 args, changed 1 rows
2024-04-26 14:10:10,429 [TRACE] BaseMappedStatement delete arguments: [1]
2024-04-26 14:10:10,429 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,429 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,430 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,430 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,430 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,432 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,432 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,432 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,433 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,433 [TRACE] BaseMappedStatement insert arguments: [1, 0, null]
2024-04-26 14:10:10,433 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,434 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,434 [TRACE] BaseMappedStatement insert arguments: [2, 0, null]
2024-04-26 14:10:10,434 [DEBUG] BaseMappedStatement assigned id '2' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,434 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,434 [TRACE] BaseMappedStatement insert arguments: [3, 0, null]
2024-04-26 14:10:10,434 [DEBUG] BaseMappedStatement assigned id '3' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,437 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,438 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,438 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,439 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,440 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,440 [TRACE] BaseMappedStatement insert arguments: [13131511, 0, null]
2024-04-26 14:10:10,440 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,440 [DEBUG] BaseMappedStatement delete data with statement 'DELETE FROM `foo` WHERE `id` = ?' and 1 args, changed 1 rows
2024-04-26 14:10:10,440 [TRACE] BaseMappedStatement delete arguments: [1]
2024-04-26 14:10:10,440 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got no results
2024-04-26 14:10:10,440 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,441 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `foo` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,441 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,443 [INFO] TableUtils dropping table 'versionfielddatestring'
2024-04-26 14:10:10,444 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "VERSIONFIELDDATESTRING" not found; SQL statement:
DROP TABLE `versionfielddatestring`  [42102-200]' for statement: DROP TABLE `versionfielddatestring` 
2024-04-26 14:10:10,444 [INFO] TableUtils creating table 'versionfielddatestring'
2024-04-26 14:10:10,445 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `versionfielddatestring` (`id` INTEGER AUTO_INCREMENT , `stuff1` VARCHAR(255) , `version` VARCHAR(50) , `stuff2` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,446 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `versionfielddatestring` (`stuff1` ,`version` ,`stuff2` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,446 [TRACE] BaseMappedStatement insert arguments: [null, 2024-04-26 14:10:10.000445, null]
2024-04-26 14:10:10,446 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in VersionFieldDateString object
2024-04-26 14:10:10,448 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfielddatestring` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,448 [TRACE] BaseMappedStatement update arguments: [null, 2024-04-26 14:10:10.000447, null, 1, 2024-04-26 14:10:10.000445]
2024-04-26 14:10:10,449 [INFO] TableUtils dropping table 'versionfield'
2024-04-26 14:10:10,450 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "VERSIONFIELD" not found; SQL statement:
DROP TABLE `versionfield`  [42102-200]' for statement: DROP TABLE `versionfield` 
2024-04-26 14:10:10,451 [INFO] TableUtils creating table 'versionfield'
2024-04-26 14:10:10,451 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `versionfield` (`id` INTEGER AUTO_INCREMENT , `stuff1` VARCHAR(255) , `version` INTEGER , `stuff2` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,452 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `versionfield` (`stuff1` ,`version` ,`stuff2` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,452 [TRACE] BaseMappedStatement insert arguments: [null, 10, null]
2024-04-26 14:10:10,452 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in VersionField object
2024-04-26 14:10:10,452 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `versionfield` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,452 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,453 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfield` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,453 [TRACE] BaseMappedStatement update arguments: [null, 11, null, 1, 10]
2024-04-26 14:10:10,453 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfield` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,453 [TRACE] BaseMappedStatement update arguments: [null, 12, null, 1, 11]
2024-04-26 14:10:10,454 [INFO] TableUtils dropping table 'versionfielddate'
2024-04-26 14:10:10,455 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "VERSIONFIELDDATE" not found; SQL statement:
DROP TABLE `versionfielddate`  [42102-200]' for statement: DROP TABLE `versionfielddate` 
2024-04-26 14:10:10,455 [INFO] TableUtils creating table 'versionfielddate'
2024-04-26 14:10:10,456 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `versionfielddate` (`id` INTEGER AUTO_INCREMENT , `stuff1` VARCHAR(255) , `version` TIMESTAMP , `stuff2` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,456 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `versionfielddate` (`stuff1` ,`version` ,`stuff2` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,456 [TRACE] BaseMappedStatement insert arguments: [null, 2024-04-26 14:10:10.456, null]
2024-04-26 14:10:10,456 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in VersionFieldDate object
2024-04-26 14:10:10,457 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfielddate` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,457 [TRACE] BaseMappedStatement update arguments: [null, 2024-04-26 14:10:10.457, null, 1, 2024-04-26 14:10:10.456]
2024-04-26 14:10:10,458 [INFO] TableUtils dropping table 'versionfielddatelong'
2024-04-26 14:10:10,459 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "VERSIONFIELDDATELONG" not found; SQL statement:
DROP TABLE `versionfielddatelong`  [42102-200]' for statement: DROP TABLE `versionfielddatelong` 
2024-04-26 14:10:10,459 [INFO] TableUtils creating table 'versionfielddatelong'
2024-04-26 14:10:10,459 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `versionfielddatelong` (`id` INTEGER AUTO_INCREMENT , `stuff1` VARCHAR(255) , `version` BIGINT , `stuff2` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,460 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `versionfielddatelong` (`stuff1` ,`version` ,`stuff2` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,460 [TRACE] BaseMappedStatement insert arguments: [null, 1714137010459, null]
2024-04-26 14:10:10,460 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in VersionFieldDateLong object
2024-04-26 14:10:10,460 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfielddatelong` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,460 [TRACE] BaseMappedStatement update arguments: [null, 1714137010460, null, 1, 1714137010459]
2024-04-26 14:10:10,461 [INFO] TableUtils dropping table 'versionfield'
2024-04-26 14:10:10,462 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "VERSIONFIELD" not found; SQL statement:
DROP TABLE `versionfield`  [42102-200]' for statement: DROP TABLE `versionfield` 
2024-04-26 14:10:10,462 [INFO] TableUtils creating table 'versionfield'
2024-04-26 14:10:10,462 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `versionfield` (`id` INTEGER AUTO_INCREMENT , `stuff1` VARCHAR(255) , `version` INTEGER , `stuff2` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,463 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `versionfield` (`stuff1` ,`version` ,`stuff2` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,463 [TRACE] BaseMappedStatement insert arguments: [null, 0, null]
2024-04-26 14:10:10,463 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in VersionField object
2024-04-26 14:10:10,463 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfield` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,463 [TRACE] BaseMappedStatement update arguments: [null, 1, null, 1, 0]
2024-04-26 14:10:10,463 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfield` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,464 [TRACE] BaseMappedStatement update arguments: [null, 2, null, 1, 1]
2024-04-26 14:10:10,464 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `versionfield` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,464 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,464 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfield` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 1 rows
2024-04-26 14:10:10,464 [TRACE] BaseMappedStatement update arguments: [null, 3, null, 1, 2]
2024-04-26 14:10:10,464 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `versionfield` SET `stuff1` = ?, `version` = ?, `stuff2` = ? WHERE `id` = ? AND `version` = ?' and 5 args, changed 0 rows
2024-04-26 14:10:10,464 [TRACE] BaseMappedStatement update arguments: [null, 3, null, 1, 2]
2024-04-26 14:10:10,465 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,466 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,466 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,467 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,467 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,467 [TRACE] BaseMappedStatement insert arguments: [1, 10, null]
2024-04-26 14:10:10,467 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,467 [DEBUG] StatementExecutor executing raw query for: SELECT * FROM `foo` WHERE `val` = ?
2024-04-26 14:10:10,467 [TRACE] StatementExecutor query arguments: [1]
2024-04-26 14:10:10,468 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,469 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,469 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,470 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,471 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,471 [TRACE] BaseMappedStatement insert arguments: [342, 0, null]
2024-04-26 14:10:10,471 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,471 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,471 [TRACE] BaseMappedStatement insert arguments: [9045342, 0, null]
2024-04-26 14:10:10,471 [DEBUG] BaseMappedStatement assigned id '2' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,471 [DEBUG] StatementExecutor executing raw query for: SELECT MAX(val) FROM `foo`
2024-04-26 14:10:10,476 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,476 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,477 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,477 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,478 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,478 [TRACE] BaseMappedStatement insert arguments: [1, 10, null]
2024-04-26 14:10:10,478 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,478 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,478 [TRACE] BaseMappedStatement insert arguments: [10, 5, null]
2024-04-26 14:10:10,478 [DEBUG] BaseMappedStatement assigned id '2' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,478 [DEBUG] StatementExecutor executing raw query for: SELECT COUNT(*) FROM `foo`
2024-04-26 14:10:10,478 [DEBUG] StatementExecutor executing raw query for: SELECT MIN(val), MAX(val) FROM `foo`
2024-04-26 14:10:10,479 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,480 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,480 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,481 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,481 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,481 [TRACE] BaseMappedStatement insert arguments: [1213213, 0, null]
2024-04-26 14:10:10,481 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,482 [DEBUG] StatementExecutor executing raw query for: select id as SOME_ID, val as SOME_VAL from foo
2024-04-26 14:10:10,483 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,484 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,484 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,484 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,485 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,485 [TRACE] BaseMappedStatement insert arguments: [243342, 0, null]
2024-04-26 14:10:10,485 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,485 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,485 [TRACE] BaseMappedStatement insert arguments: [243342, 0, null]
2024-04-26 14:10:10,485 [DEBUG] BaseMappedStatement assigned id '2' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,485 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`val` ,`equal` ,`string` ) VALUES (?,?,?)' and 3 args, changed 1 rows
2024-04-26 14:10:10,485 [TRACE] BaseMappedStatement insert arguments: [6543, 0, null]
2024-04-26 14:10:10,485 [DEBUG] BaseMappedStatement assigned id '3' from keyholder to 'id' in Foo object
2024-04-26 14:10:10,485 [DEBUG] StatementExecutor executing raw query for: SELECT `val` FROM `foo` GROUP BY `val` HAVING COUNT(VAL) > 1
2024-04-26 14:10:10,487 [DEBUG] StatementExecutor executing raw query for: SELECT `val` FROM `foo` GROUP BY `val` HAVING COUNT(VAL) > 2
2024-04-26 14:10:10,487 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,489 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,489 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,489 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`id` INTEGER AUTO_INCREMENT , `val` INTEGER , `equal` INTEGER , `string` VARCHAR(255) , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,490 [ERROR] Log4jLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testErrorStringThrowable(BaseLogBackendTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,490 [WARN] Log4jLogBackendTest message
2024-04-26 14:10:10,490 [INFO] Log4jLogBackendTest hello there
2024-04-26 14:10:10,490 [ERROR] Log4jLogBackendTest message
2024-04-26 14:10:10,491 [INFO] Log4jLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testInfoStringThrowable(BaseLogBackendTest.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,491 [INFO] Log4jLogBackendTest message
2024-04-26 14:10:10,491 [FATAL] Log4jLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testFatalStringThrowable(BaseLogBackendTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,491 [FATAL] Log4jLogBackendTest message
2024-04-26 14:10:10,492 [WARN] Log4jLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testWarnStringThrowable(BaseLogBackendTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,493 [DEBUG] LocalLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testDebugStringThrowable(BaseLogBackendTest.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
IO exception reading the log properties file '/ormliteLocalLog.properties': java.io.IOException: simulated exception
2024-04-26 14:10:10,494 [DEBUG] LocalLogBackendTest message
2024-04-26 14:10:10,494 [ERROR] LocalLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testErrorStringThrowable(BaseLogBackendTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,494 [INFO] LocalLogBackendTest message
2024-04-26 14:10:10,495 [FATAL] LocalLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testFatalStringThrowable(BaseLogBackendTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,495 [FATAL] LocalLogBackendTest message
2024-04-26 14:10:10,498 [INFO] LocalLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testInfoStringThrowable(BaseLogBackendTest.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
2024-04-26 14:10:10,498 [INFO] LocalLogBackendTest hello there
2024-04-26 14:10:10,498 [WARNING] LocalLogBackendTest message
java.lang.Throwable: log throwable
	at com.j256.ormlite.logger.BaseLogBackendTest.testWarnStringThrowable(BaseLogBackendTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
Line is not in the format of 'pattern = level': x
Level ' INVALID_LEVEL' was not found
Line is not in the format of 'pattern = level': x
2024-04-26 14:10:10,500 [ERROR] LocalLogBackendTest message
2024-04-26 14:10:10,500 [WARNING] LocalLogBackendTest message
Skipping unknown value class class java.lang.Class
Skipping unknown value class class com.j256.ormlite.field.DataType
2024-04-26 14:10:10,504 [WARNING] LoggerFactoryTest Unable to create instance of class com.j256.ormlite.android.AndroidLogBackend$AndroidLogBackendFactory for log type ANDROID, using local log: com.j256.ormlite.android.AndroidLogBackend$AndroidLogBackendFactory
2024-04-26 14:10:10,505 [WARNING] LoggerFactory Could not find valid log-type from system property 'com.j256.simplelogger.backend', value 'some.wrong.class'
2024-04-26 14:10:10,509 [INFO] TableUtils dropping table 'serializedupdate'
2024-04-26 14:10:10,509 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "SERIALIZEDUPDATE" not found; SQL statement:
DROP TABLE `serializedupdate`  [42102-200]' for statement: DROP TABLE `serializedupdate` 
2024-04-26 14:10:10,510 [INFO] TableUtils creating table 'serializedupdate'
2024-04-26 14:10:10,510 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `serializedupdate` (`id` INTEGER AUTO_INCREMENT , `serialized` BLOB , PRIMARY KEY (`id`) ) 
2024-04-26 14:10:10,512 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `serializedupdate` (`serialized` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,512 [TRACE] BaseMappedStatement insert arguments: [[-84, -19, 0, 5, 115, 114, 0, 65, 99, 111, 109, 46, 106, 50, 53, 54, 46, 111, 114, 109, 108, 105, 116, 101, 46, 102, 105, 101, 108, 100, 46, 116, 121, 112, 101, 115, 46, 83, 101, 114, 105, 97, 108, 105, 122, 97, 98, 108, 101, 84, 121, 112, 101, 84, 101, 115, 116, 36, 83, 101, 114, 105, 97, 108, 105, 122, 101, 100, 70, 105, 101, 108, 100, 62, -28, 13, -79, 82, -70, -126, 120, 2, 0, 1, 76, 0, 3, 102, 111, 111, 116, 0, 18, 76, 106, 97, 118, 97, 47, 108, 97, 110, 103, 47, 83, 116, 114, 105, 110, 103, 59, 120, 112, 116, 0, 3, 119, 111, 119]]
2024-04-26 14:10:10,512 [DEBUG] BaseMappedStatement assigned id '1' from keyholder to 'id' in SerializedUpdate object
2024-04-26 14:10:10,515 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `serializedupdate` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,515 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,515 [DEBUG] BaseMappedStatement update data with statement 'UPDATE `serializedupdate` SET `serialized` = ? WHERE `id` = ?' and 2 args, changed 1 rows
2024-04-26 14:10:10,515 [TRACE] BaseMappedStatement update arguments: [[-84, -19, 0, 5, 115, 114, 0, 65, 99, 111, 109, 46, 106, 50, 53, 54, 46, 111, 114, 109, 108, 105, 116, 101, 46, 102, 105, 101, 108, 100, 46, 116, 121, 112, 101, 115, 46, 83, 101, 114, 105, 97, 108, 105, 122, 97, 98, 108, 101, 84, 121, 112, 101, 84, 101, 115, 116, 36, 83, 101, 114, 105, 97, 108, 105, 122, 101, 100, 70, 105, 101, 108, 100, 62, -28, 13, -79, 82, -70, -126, 120, 2, 0, 1, 76, 0, 3, 102, 111, 111, 116, 0, 18, 76, 106, 97, 118, 97, 47, 108, 97, 110, 103, 47, 83, 116, 114, 105, 110, 103, 59, 120, 112, 116, 0, 3, 122, 105, 112], 1]
2024-04-26 14:10:10,516 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `serializedupdate` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,516 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,517 [DEBUG] BaseMappedStatement prepared statement 'UPDATE `serializedupdate` SET `serialized` = ? WHERE `id` = 1' with 1 args
2024-04-26 14:10:10,517 [TRACE] BaseMappedStatement prepared statement arguments: [[-84, -19, 0, 5, 115, 114, 0, 65, 99, 111, 109, 46, 106, 50, 53, 54, 46, 111, 114, 109, 108, 105, 116, 101, 46, 102, 105, 101, 108, 100, 46, 116, 121, 112, 101, 115, 46, 83, 101, 114, 105, 97, 108, 105, 122, 97, 98, 108, 101, 84, 121, 112, 101, 84, 101, 115, 116, 36, 83, 101, 114, 105, 97, 108, 105, 122, 101, 100, 70, 105, 101, 108, 100, 62, -28, 13, -79, 82, -70, -126, 120, 2, 0, 1, 76, 0, 3, 102, 111, 111, 116, 0, 18, 76, 106, 97, 118, 97, 47, 108, 97, 110, 103, 47, 83, 116, 114, 105, 110, 103, 59, 120, 112, 116, 0, 5, 99, 114, 97, 99, 107]]
2024-04-26 14:10:10,518 [DEBUG] BaseMappedStatement query-for-id using 'SELECT * FROM `serializedupdate` WHERE `id` = ?' and 1 args, got 1 result
2024-04-26 14:10:10,518 [TRACE] BaseMappedStatement query-for-id arguments: [1]
2024-04-26 14:10:10,519 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,520 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,520 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,520 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`serializable` BLOB ) 
2024-04-26 14:10:10,520 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`serializable` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,520 [TRACE] BaseMappedStatement insert arguments: [null]
2024-04-26 14:10:10,521 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,522 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,522 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,522 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`serializable` BLOB ) 
2024-04-26 14:10:10,523 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`serializable` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,523 [TRACE] BaseMappedStatement insert arguments: [[-84, -19, 0, 5, 115, 114, 0, 17, 106, 97, 118, 97, 46, 108, 97, 110, 103, 46, 73, 110, 116, 101, 103, 101, 114, 18, -30, -96, -92, -9, -127, -121, 56, 2, 0, 1, 73, 0, 5, 118, 97, 108, 117, 101, 120, 114, 0, 16, 106, 97, 118, 97, 46, 108, 97, 110, 103, 46, 78, 117, 109, 98, 101, 114, -122, -84, -107, 29, 11, -108, -32, -117, 2, 0, 0, 120, 112, 79, 90, -120, 11]]
2024-04-26 14:10:10,524 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,524 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
2024-04-26 14:10:10,525 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,526 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,526 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,526 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`serializable` BLOB ) 
2024-04-26 14:10:10,526 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`serializable` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,526 [TRACE] BaseMappedStatement insert arguments: [null]
2024-04-26 14:10:10,527 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,527 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
2024-04-26 14:10:10,527 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,528 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,528 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,528 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`byteField` BLOB ) 
2024-04-26 14:10:10,528 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`byteField` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,528 [TRACE] BaseMappedStatement insert arguments: [[1, 2, 3, 4, 5]]
2024-04-26 14:10:10,530 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,531 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,531 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,531 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`floatField` FLOAT ) 
2024-04-26 14:10:10,532 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`floatField` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,532 [TRACE] BaseMappedStatement insert arguments: [1331.221]
2024-04-26 14:10:10,533 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,533 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
2024-04-26 14:10:10,533 [INFO] TableUtils dropping table 'foo'
2024-04-26 14:10:10,534 [INFO] TableUtils ignoring drop error 'org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FOO" not found; SQL statement:
DROP TABLE `foo`  [42102-200]' for statement: DROP TABLE `foo` 
2024-04-26 14:10:10,534 [INFO] TableUtils creating table 'foo'
2024-04-26 14:10:10,535 [INFO] TableUtils executed create table statement changed 0 rows: CREATE TABLE `foo` (`floatField` FLOAT ) 
2024-04-26 14:10:10,535 [DEBUG] BaseMappedStatement insert data with statement 'INSERT INTO `foo` (`floatField` ) VALUES (?)' and 1 args, changed 1 rows
2024-04-26 14:10:10,535 [TRACE] BaseMappedStatement insert arguments: [null]
2024-04-26 14:10:10,535 [DEBUG] BaseMappedStatement prepared statement 'SELECT * FROM `foo`' with 0 args
2024-04-26 14:10:10,535 [DEBUG] StatementExecutor query of 'SELECT * FROM `foo`' with 0 args returned 1 results
java.lang.Exception: Unexpected exception, expected<java.lang.RuntimeException> but was<java.lang.AssertionError>
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:30)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
Caused by: java.lang.AssertionError: 
  Unexpected method call LogBackend.isLevelEnabled(DEBUG):
	at org.easymock.internal.MockInvocationHandler.invoke(MockInvocationHandler.java:44)
	at org.easymock.internal.ObjectMethodsFilter.invoke(ObjectMethodsFilter.java:94)
	at com.sun.proxy.$Proxy13.isLevelEnabled(Unknown Source)
	at com.j256.ormlite.logger.Logger.logIfEnabled(Logger.java:659)
	at com.j256.ormlite.logger.Logger.log(Logger.java:610)
	at com.j256.ormlite.dao.RuntimeExceptionDao.logMessage(RuntimeExceptionDao.java:981)
	at com.j256.ormlite.dao.RuntimeExceptionDao.delete(RuntimeExceptionDao.java:354)
	at com.j256.ormlite.dao.RuntimeExceptionDaoTest.testDeleteThrow(RuntimeExceptionDaoTest.java:610)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
	... 34 more
java.lang.Exception: Unexpected exception, expected<java.lang.RuntimeException> but was<java.lang.AssertionError>
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:30)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
Caused by: java.lang.AssertionError: 
  Unexpected method call LogBackend.isLevelEnabled(DEBUG):
	at org.easymock.internal.MockInvocationHandler.invoke(MockInvocationHandler.java:44)
	at org.easymock.internal.ObjectMethodsFilter.invoke(ObjectMethodsFilter.java:94)
	at com.sun.proxy.$Proxy13.isLevelEnabled(Unknown Source)
	at com.j256.ormlite.logger.Logger.logIfEnabled(Logger.java:659)
	at com.j256.ormlite.logger.Logger.log(Logger.java:610)
	at com.j256.ormlite.dao.RuntimeExceptionDao.logMessage(RuntimeExceptionDao.java:981)
	at com.j256.ormlite.dao.RuntimeExceptionDao.rollBack(RuntimeExceptionDao.java:898)
	at com.j256.ormlite.dao.RuntimeExceptionDaoTest.testRollbackThrows(RuntimeExceptionDaoTest.java:1074)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
	... 34 more
java.lang.Exception: Unexpected exception, expected<java.lang.RuntimeException> but was<java.lang.AssertionError>
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:30)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
Caused by: java.lang.AssertionError: 
  Unexpected method call LogBackend.isLevelEnabled(DEBUG):
	at org.easymock.internal.MockInvocationHandler.invoke(MockInvocationHandler.java:44)
	at org.easymock.internal.ObjectMethodsFilter.invoke(ObjectMethodsFilter.java:94)
	at com.sun.proxy.$Proxy13.isLevelEnabled(Unknown Source)
	at com.j256.ormlite.logger.Logger.logIfEnabled(Logger.java:659)
	at com.j256.ormlite.logger.Logger.log(Logger.java:610)
	at com.j256.ormlite.dao.RuntimeExceptionDao.logMessage(RuntimeExceptionDao.java:981)
	at com.j256.ormlite.dao.RuntimeExceptionDao.queryForFieldValuesArgs(RuntimeExceptionDao.java:187)
	at com.j256.ormlite.dao.RuntimeExceptionDaoTest.testQueryForFieldsValuesArgsThrow(RuntimeExceptionDaoTest.java:478)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
	... 34 more
java.lang.Exception: Unexpected exception, expected<java.lang.RuntimeException> but was<java.lang.AssertionError>
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:30)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
Caused by: java.lang.AssertionError: 
  Unexpected method call LogBackend.isLevelEnabled(DEBUG):
	at org.easymock.internal.MockInvocationHandler.invoke(MockInvocationHandler.java:44)
	at org.easymock.internal.ObjectMethodsFilter.invoke(ObjectMethodsFilter.java:94)
	at com.sun.proxy.$Proxy13.isLevelEnabled(Unknown Source)
	at com.j256.ormlite.logger.Logger.logIfEnabled(Logger.java:659)
	at com.j256.ormlite.logger.Logger.log(Logger.java:610)
	at com.j256.ormlite.dao.RuntimeExceptionDao.logMessage(RuntimeExceptionDao.java:981)
	at com.j256.ormlite.dao.RuntimeExceptionDao.iterator(RuntimeExceptionDao.java:472)
	at com.j256.ormlite.dao.RuntimeExceptionDaoTest.testIteratorThrow(RuntimeExceptionDaoTest.java:677)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
	... 34 more
java.lang.Exception: Unexpected exception, expected<java.lang.RuntimeException> but was<java.lang.AssertionError>
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:30)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.execute(JUnitTestExecutor.java:259)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.executeWithJUnit4Runner(JUnitTestExecutor.java:327)
	at edu.illinois.cs.testrunner.execution.JUnitTestExecutor.runOrder(JUnitTestExecutor.java:49)
	at edu.illinois.cs.testrunner.execution.Executor$.$anonfun$run$1(Executor.scala:30)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.Try$.apply(Try.scala:209)
	at edu.illinois.cs.testrunner.execution.Executor$.run(Executor.scala:28)
	at edu.illinois.cs.testrunner.execution.Executor$.main(Executor.scala:18)
	at edu.illinois.cs.testrunner.execution.Executor.main(Executor.scala)
Caused by: java.lang.AssertionError: 
  Unexpected method call LogBackend.isLevelEnabled(DEBUG):
	at org.easymock.internal.MockInvocationHandler.invoke(MockInvocationHandler.java:44)
	at org.easymock.internal.ObjectMethodsFilter.invoke(ObjectMethodsFilter.java:94)
	at com.sun.proxy.$Proxy13.isLevelEnabled(Unknown Source)
	at com.j256.ormlite.logger.Logger.logIfEnabled(Logger.java:659)
	at com.j256.ormlite.logger.Logger.log(Logger.java:610)
	at com.j256.ormlite.dao.RuntimeExceptionDao.logMessage(RuntimeExceptionDao.java:981)
	at com.j256.ormlite.dao.RuntimeExceptionDao.queryRaw(RuntimeExceptionDao.java:498)
	at com.j256.ormlite.dao.RuntimeExceptionDaoTest.testQueryRawThrow(RuntimeExceptionDaoTest.java:721)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
	... 34 more
